modules/administration-guide/pages/enabling-users-to-run-multiple-workspaces-simultaneously.adoc

```diff
@@ -24,19 +24,24 @@ spec:
 
 .Procedure
 
-. Get the name of the {prod-short} namespace:
+. Get the name of the {prod-short} namespace. The default is `{prod-namespace}`.
 +
 [source,terminal,subs="+quotes,attributes"]
 ----
 $ {orch-cli} get checluster --all-namespaces \
   -o=jsonpath="{.items[*].metadata.namespace}"
 ----
-+
-include::example$snip_{project-context}-default-namespace.adoc[]
 
 . Configure the `runningLimit`:
 +
-include::example$snip_{project-context}-enabling-users-to-run-multiple-workspaces-simultaneously.adoc[]
+[source,subs="+quotes,attributes"]
```

Comment:
```suggestion
[source,terminal,subs="+quotes,attributes"]
```

Reply:
To sum up Robert's feedback, we don't use `source,terminal` because of these reasons:
1. The tooltip `Terminal` is confusing to readers because `Terminal` is not a language.
2. Regarding the convenience of the copy-paste tooltip button: We write `$` at the beginning of commands, so `$` also gets copy-pasted, requiring a manual edit of the copy-pasted text, as opposed to simply selecting it with a mouse and pasting it. This can mess things up when users paste it onto a command line, which is where indeed they should be pasting it.
3. Regarding the convenience of the copy-paste tooltip button: the copy-paste tooltip button is only a feature in the Che docs, and it is absent from the Dev Spaces docs.